source/documentation/fuseki2/fuseki-main.md

```diff
@@ -33,201 +33,38 @@ persistent work area on-disk.
 the server can also be run as:
 
     java -cp jena-fuseki-server-$VER.jar:...OtherJars... \
         org.apache.jena.fuseki.main.cmds.FusekiMainCmd ARGS
 
+## Docker {#fuseki-docker}
 
-## Application Use {#usage}
-
-The application can safely access and modify the data published by the server if it does
-so inside a [transaction](/documentation/txn/) using an appropriate
-storage choice. `DatasetFactory.createTxnMem()` is a good choice for in-memory use;
-[TDB](/documentation/tdb/) is a good choice for a persistent database.
-
-To build and start the server:
-
-    Dataset ds = ...
-    FusekiServer server = FusekiServer.create()
-      .add("/rdf", ds)
-      .build() ;
-    server.start() ;
-
-then the application can modify the dataset:
-
-    // Add some data while live.
-    // Write transaction.
-    Txn.execWrite(dsg, ()->RDFDataMgr.read(dsg, "D.trig")) ;
-
-or read the dataset and see any updates made by remote systems:
-
-    // Query data while live
-    // Read transaction.
-    Txn.execRead(dsg, ()->{
-    Dataset ds = DatasetFactory.wrap(dsg) ;
-    try (QueryExecution qExec = QueryExecutionFactory.create("SELECT * { ?s  ?o}", ds) ) {
-        ResultSet rs = qExec.execSelect() ;
-        ResultSetFormatter.out(rs) ;
-      }
-    }) ;
-
-The full Jena API can be used provided operations (read and write) are inside
-a transaction.
-
-## Dependencies and Setup {#dependencies}
-
-To include an embedded Fuseki server in the application:
-
-    <dependency>
-      <groupId>org.apache.jena</groupId>
-      <artifactId>jena-fuseki-main</artifactId>
-      <version>3.x.y</version> <!-- Set the version -->
-    </dependency>
-
-This brings in enough dependencies to run Fuseki. Application writers are strongly
-encouraged to use a dependency manager because the number of Jetty and other dependencies
-is quite large and difficult to set manually.
-
-This dependency does not include a logging setting. Fuseki uses [slf4j](http://slf4j.org/).
-See section "[Logging](#logging)" for details.
-
-If the application wishes to use a dataset with a [text-index](/documentation/query/text-query.html)
-then the application will also need to include jena-text in its dependencies.
-
-## Logging {#logging}
-
-The application must set the logging provided for [slf4j](http://slf4j.org/).
-Apache Jena provides helpers for the JDK-provided java logging and for Apache Log4j v2.
-
-Using the JDK-provided java logging:
-
-    LogCtl.setJavaLogging()
-
-and a dependency of:
-
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-jdk14</artifactId>
-      <version>1.x.y</version>
-    </dependency>
-
-For Apache Log4j2: - this is how the full Fuseki server sets its logging:
+A kit to build a container with docker or docker compose 
 
-    FusekiLogging.setLogging();
+    https://repo1.maven.org/maven2/org/apache/jena/jena-fuseki-docker/
 
-and dependency:
+Note; take care thet databases are on mounted volumes if they are to peristent
```

Comment:
s/thet/that

s/they are to persistent/they are to be persistent (or they are to persist?)